eif_defs/src/lib.rs

```diff
@@ -18,6 +18,10 @@ use std::mem::size_of;
 
 pub const EIF_MAGIC: [u8; 4] = [46, 101, 105, 102]; // .eif in ascii
 pub const MAX_NUM_SECTIONS: usize = 32;
+/// EIF Header Flags
+/// bits 1  : Architecture - toggled for aarch64, cleared for x86_64
+/// bits 2-8: Unused
+pub const EIF_HDR_ARCH_ARM64: u16 = 0x1;
```

Comment:
What if we ever need add RISC-V or something else? We should have a separate arch field.

Reply:
Given that we might not want to add another version of EIF right now, I think this is fine.